lib/formatters/tapFormatter.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * @type {import('stylelint').Formatter}
+ */
+const tapFormatter = (results) => {
+	let lines = [`TAP version 13\n1..${results.length}`];
+
+	results.forEach((result, index) => {
+		lines.push(
+			`${result.errored ? 'not ok' : 'ok'} ${index + 1} - ${result.ignored ? 'ignored ' : ''}${
+				result.source
+			}`,
+		);
+
+		if (result.warnings.length > 0) {
+			lines.push(...['---', 'messages:']);
+
+			result.warnings.forEach((warning) => {
+				lines.push(
```

Comment:
you can use `N` arguments to push in Array.prototype.push https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
Great point! Updated